src/jaxsim/rbda/contacts/relaxed_rigid.py

```diff
@@ -499,6 +514,7 @@ def _regularizers(
         penetration: jtp.Array,
         velocity: jtp.Array,
         parameters: RelaxedRigidContactsParams,
+        indices_of_enabled_collidable_points: npt.NDArray,
```

Comment:
Also in this case I think that I need a static numpy array to slice a Jax array, otherwise we'll incur into Jax errors due to indexing with traced arrays in jit compiled functions.